app.mjs

```diff
@@ -29,6 +30,21 @@ const chainIdParamSchema = {
  * @param {import("fastify").FastifyInstance} fastify
  */
 export async function server(fastify, opts) {
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    environment: process.env.NODE_ENV || "development"
+  });
+  fastify.setErrorHandler(function(err, request, reply) {
+    Sentry.withScope(scope => {
+        if (request.raw.ip) {
+          scope.setUser({
+            ip_address: request.raw.ip
+          });
+        };
+        scope.setTag("path", request.raw.url);
+        Sentry.captureException(err);
+    });
```

Comment:
Looks good. I think we just need to add a `reply.send(error)` here so it actually returns a 5xx response. As it is right now, it actually returns a 400 response.

This is going to throw off our monitoring tools